packages/h5wasm/src/h5wasm-api.test.ts

```diff
@@ -1,22 +1,19 @@
+import { existsSync } from 'node:fs';
+import { readFile } from 'node:fs/promises';
+import path from 'node:path';
+
+import { DataProviderApi, getValueOrError } from '@h5web/app';
 import {
   assertDataset,
   assertGroup,
   assertGroupWithChildren,
   hasNonNullShape,
 } from '@h5web/shared/guards';
-import type {
-  ArrayShape,
-  Dataset,
-  ScalarShape,
-} from '@h5web/shared/hdf5-models';
-import { existsSync } from 'fs';
-import { readFile } from 'fs/promises';
-import path from 'path';
 import { expect, test } from 'vitest';
 
 import { H5WasmApi } from './h5wasm-api';
 
-const LOCAL_TEST_FILE = path.resolve(__dirname, '../dist-h5/sample.h5');
+const LOCAL_TEST_FILE = path.resolve(process.cwd(), 'dist/sample.h5');
```

Comment:
I've moved the Python script to the root of the monorepo. The sample file is now generated into a `dist` folder also at the root. I've considered moving it to the shared package, but it didn't feel right, because the sample file is only imported from `@h5web/h5wasm` (and only when it exists, otherwise it falls back to fetching it from silx.org).

I preferred to make it clear that the script and sample file are part of the set up of the repository as a whole and not of a specific project.